tests_e2e/lisa/testsuites/agent-bvt.py

```diff
@@ -22,20 +18,17 @@
     """,
     requirement=simple_requirement(unsupported_os=[]),
 )
-class AgentBvt(TestSuite):
```

Comment:
Given the limitations on LISA's TestSuite, now I am modeling a DCR scenario as a TestSuite with a single test (main), which invokes each of the steps in the scenario.